example/otel-collector/go.sum

```diff
@@ -4,6 +4,8 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/antihax/optional v1.0.0/go.mod h1:uupD/76wgC+ih3iEmQUL+0Ugr19nfwCT1kdvxnR2qWY=
 github.com/benbjohnson/clock v1.0.3 h1:vkLuvpK4fmtSCuo60+yC63p7y0BmQ8gm5ZXGuBCJyXg=
 github.com/benbjohnson/clock v1.0.3/go.mod h1:bGMdMPoPVvcYyt1gHDf4J2KE153Yf9BuiUKYMaxlTDM=
+github.com/cenkalti/backoff/v4 v4.1.0 h1:c8LkOFQTzuO0WBM/ae5HdGQuZPfPxp7lqBRwQRm4fSc=
```

Comment:
are we bringing a dependency while only https://github.com/cenkalti/backoff/blob/v4/exponential.go is probably what we need?

References:
- from https://go-proverbs.github.io/:
> A little copying is better than a little dependency.

- https://research.swtch.com/deps

Reply:
+1 for importing this and not just copying. The dependency is not that large and the algorithm it implements is complex and needs maintenance. I would not be in favor of optimizing our dependencies at this point, especially since we are importing the grpc and protobuf libraries.